data/hai/hai reveal 1 intro.txt

```diff
@@ -524,12 +532,26 @@ mission "Hai Leaks Response 1"
 			`	You haven't really seen very much of the Hai so far, and you can't help but think the size of the detachment is rather militant of them. It's impossible to judge if it's for your safety or whether, given your status as a "war hero", it's for theirs.`
 			`	Word of your status in human space also seems to have spread ahead of you, perhaps because you are a significant human visitor in the current climate. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
 			label resume
+			choice
+				`	(Keep silent and observe.)`
+					goto observe
+				`	(Ask about the large security detachment.)`
```

Comment:
```suggestion
				`	"Why do I have so many escorts?"`
```

Reply:
That doesn't look right in-game because there isn't anyone to ask. I am pondering how to modify the prior text.

Reply:
You could mention the Hai officer when the player first sees the detachment, something like "As you arrive on Hai-home, you find that a reasonable security detachment is waiting at your designated landing berth. One Hai with a very decorated uniform comes up to greet you, and tells you that he will escort you to the government chambers."

Reply:
I changed it so the security detachment suddenly leads you to the left to avoid a big crowd.